src/main/java/formflow/library/ValidationService.java

```diff
@@ -102,6 +104,13 @@ private Map<String, List<String>> performFieldLevelValidation(String flowName, F
         key = key.replace("[]", "");
       }
 
+      String originalKey = key;
+
+      if (key.contains("_wildcard")) {
```

Comment:
Wondering if _Nth would be more appropriate here? Where Nth represents some number in an unknown total?

Not a big deal, _wildcard works, but Nth came to mind.

Reply:
We need something we could match on easily - the Nth idea would be hard to match short of regex and w/o a prefix to match one.  However, what you're suggesting sounds like a way to track the **number** of "dynamic" fields, if need be.  I could see how that could be handy in some situations.  

Like what about: "_multiField_NN_", where N is a number (0-9). 

Does make me think that the "_wildcard" should, at the very least, be a static string const.